.gitlab-ci.yml

```diff
@@ -45,181 +45,181 @@ image: 'ruby:2.4'
       - $SIMP_FULL_MATRIX
       - $SIMP_IPA_TEST
 
-default_el6-puppet5:
+default_el6-puppet6:
   stage: integration
-  image: 'ruby:2.4'
+  image: 'ruby:2.5'
   tags:
     - beaker
   <<: *cache_bundler
   <<: *setup_bundler_env
   variables:
-    PUPPET_VERSION: '~> 5.5'
-    BEAKER_PUPPET_COLLECTION: 'puppet5'
+    PUPPET_VERSION: '~> 6.18'
```

Comment:
According to https://puppet.com/docs/pe/2019.8/component_versions_in_recent_pe_releases.html, the PE LTS is 6.17; should we use that instead?

Reply:
@op-ct This was updated to match the package update. 6.18 fixes a bug in running `puppet apply` on the puppet server after `generate types` has been run. Without this, many of the tests will not run after the server has been configured.